src/components/layer-info.js

```diff
@@ -2,64 +2,51 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Icon from '@mapbox/mr-ui/icon';
-const slugger = require('github-slugger')();
 
 class LayerInfo extends React.PureComponent {
   render() {
-    function slug(string) {
-      slugger.reset();
-      return slugger.slug(string);
-    }
     return (
-      <h3
-        className="txt-code bg-transparent ml-neg6"
-        id={slug(this.props.name)}
-      >
-        <a
-          className="mr18 unprose color-blue-on-hover"
-          href={`#${slug(this.props.name)}`}
-        >
-          <span className="mr18">{this.props.name}</span>
-        </a>
+      <div className="mb24">
         <span
           className={
             this.props.type.indexOf('point') > -1
-              ? 'color-gray'
-              : 'color-gray-light'
+              ? 'color-blue'
+              : 'color-blue-light'
           }
         >
           <Icon name="marker" inline={true} size={32} />
```

Comment:
I recommend changing the size to `24` - this will make the icons the same size, but remove extra spacing. It will also help keep the content lined up on the left.

```suggestion
          <Icon name="marker" inline={true} size={24} />
```